addons/im_livechat/__manifest__.py

```diff
@@ -122,6 +119,9 @@
             ('remove', 'bus/static/src/simple_notification_service.js'),
             ('include', 'im_livechat.assets_embed_core'),
             'im_livechat/static/src/embed/external/**/*',
+            'web/static/src/libs/fontawesome/css/font-awesome.css',
+            'web/static/lib/odoo_ui_icons/style.css',
+            'web/static/src/scss/ui.scss',
```

Comment:
Shouldn't these be added earlier? By being at the very end of the `external` bundle, we cannot customise these style without playing with CSS specificity.

Reply:
I don't think we should override icon styles, putting them at the very end ensures they will supersede everything, which is ultimately what we want (bootstrap btn font was supplanting icons font which broke the icons).